shell_scripts/coq_platform_override_dev_pkg.sh

```diff
@@ -0,0 +1,24 @@
+#!/bin/bash
+
+###################### COPYRIGHT/COPYLEFT ######################
+
+# (C) 2020 Michael Soegtrop
+
+# Released to the public under the
+# Creative Commons CC0 1.0 Universal License
+# See https://creativecommons.org/publicdomain/zero/1.0/legalcode.txt
+
+###################### OVERRIDES #####################
+
+for override in ${COQ_PLATFORM_OVERRIDE_DEV:-}
+do
+    pkg="${override%%=*}"
+    url="${override#*=}"
+    echo "Overriding $pkg.dev url with $url"
+    mkdir -p opam/packages/$pkg/$pkg.dev/
+    opam show --raw $pkg.dev | tr -s '\n' ' ' | sed "s@ src: [^ ]*@ src: \"$url\"@" > opam/packages/$pkg/$pkg.dev/opam
```

Comment:
I think we should also replace any 2 or more spaces with just one, and make the sed line also accept no space after `:`. I could not find an easier way to "edit" an opam file field.

Reply:
I would use `sed "s@^ *src: *[^ ]*@ src: \"$url\"@"`